webpack.config.js

```diff
@@ -10,13 +10,18 @@ Encore
         "jQuery.tagsinput": "bootstrap-tagsinput"
     })
     .enableSassLoader()
-    .enableVersioning()
+    .enableVersioning(Encore.isProduction())
     .addEntry('app', './assets/js/app.js')
     .addEntry('login', './assets/js/login.js')
     .addEntry('admin', './assets/js/admin.js')
     .addEntry('search', './assets/js/search.js')
     .splitEntryChunks()
     .enableSingleRuntimeChunk()
+    .enableIntegrityHashes()
```

Comment:
should be enabled only in production. In dev, it would cause issue when using the watcher, as I'm not sure the updated `entrypoints.json` gets dumped again by the watcher (and then, that would make it an integrity violation)

Reply:
It actually works fine in both `--watch` and `dev-server` mode.

For the first one, the `entrypoints.json` does get dumped again with updated hashes, and since `--watch` implies a browser refresh the `integrity` attribute gets updated accordingly.

Here is an example with some additional logs after changing the `app.js` file:

![image](https://user-images.githubusercontent.com/850046/55581788-0ec5f300-571e-11e9-82f3-3cb2b4adaffc.png)

And with the `dev-server` enabled the integrity hashes are not computed at all since the files aren't dumped. This results in an empty `integrity` object in the `entrypoints.json` and no `integrity` attributes in the generated tags:

```js
{
  "entrypoints": {
    // (...)
    "search": {
      "js": [
        "http://localhost:8080/build/runtime.js",
        "http://localhost:8080/build/vendors~admin~app~login~search.js",
        "http://localhost:8080/build/vendors~admin~app~search.js",
        "http://localhost:8080/build/vendors~app~search.js",
        "http://localhost:8080/build/vendors~search.js",
        "http://localhost:8080/build/search.js"
      ]
    }
  },
  "integrity": {}
}
```